test/test_mkldnn.py

```diff
@@ -231,12 +237,27 @@ def test_linear(self):
         x = torch.randn(3, in_features, dtype=torch.float32) * 10
 
         for bias in [True, False]:
-            linear = torch.nn.Linear(in_features, out_features).float()
+            linear = torch.nn.Linear(in_features, out_features, bias=bias).float()
             mkldnn_linear = mkldnn_utils.to_mkldnn(copy.deepcopy(linear))
             self.assertEqual(
                 linear(x),
                 mkldnn_linear(x.to_mkldnn()).to_dense())
 
+            self._test_serialization(mkldnn_linear, (x.to_mkldnn(),))
+
+    def _test_serialization(self, module, inputs):
+        if IS_WINDOWS or IS_SANDCASTLE:
+            # TemporaryFileName is not supported in Windows and Sandcastle
+            return
+
+        with tempfile.NamedTemporaryFile() as f:
+            torch.jit.save(module, f.name)
+            f.flush()
+            loaded = torch.jit.load(f.name)
+            self.assertEqual(
```

Comment:
do you need to explicitly verify here that params are in mkldnn layout? or just passing input arg in mkldnn layout is enough?

Reply:
It's sufficient to verify it by passing mkldnn tensor as input. For modules that don't have parameters, it's not worth testing serialization anyway. Also right now there is no way to tell if a tensor is mkldnn layout at python land.